0.10/s2i/bin/assemble

```diff
@@ -7,6 +7,8 @@
 
 set -e
 
+source /opt/app-root/etc/generate_container_user
```

Comment:
maybe we should check if that script exists? iow. somebody is trying to use assemble outside this image :)